Typescript_TypeORM_Project/src/ex.ts

```diff
@@ -0,0 +1,2 @@
+// example.ts
+export const greeting = "Hello, TypeScript!";
```

Comment:
this greeting const is not being used anywhere
and what is the use of ex.ts file

Reply:
updated